airflow/stats.py

```diff
@@ -22,7 +22,6 @@
 from typing import TYPE_CHECKING, Callable
 
 from airflow.configuration import conf
-from airflow.metrics import datadog_logger, otel_logger, statsd_logger
```

Comment:
These are incredibly expensive imports, especially otel, which were slowing down the CLI parser. Import them below only where they are needed.